link

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env php
+<?php
+
+if (!is_dir(__DIR__.'/vendor/')) {
+    echo "Missing composer dependencies. Please run \"composer install\"".PHP_EOL;
+    exit(1);
+}
+
+require __DIR__.'/vendor/symfony/filesystem/Exception/ExceptionInterface.php';
+require __DIR__.'/vendor/symfony/filesystem/Exception/IOExceptionInterface.php';
+require __DIR__.'/vendor/symfony/filesystem/Exception/IOException.php';
+require __DIR__.'/vendor/symfony/filesystem/Filesystem.php';
+
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Links dependencies of a project to a local clone of the main Sylius/Stack GitHub repository.
+ * Inspired by what Kévin Dunglas <dunglas@gmail.com> did for symfony/symfony GitHub repository.
+ *
+ * @author Florian Merle <florian.david.merle@gmail.com>
```

Comment:
I think you should update that.
```
/*
 * This file is part of the Sylius package.
 *
 * (c) Sylius Sp. z o.o.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
```

Reply:
I updated the doc blocks. Since it's essentially a copy of the Symfony link file, we should probably use their copyright notice. I also added: `(c) Fabien Potencier`. What do you think we should do?"

Reply:
You should ask @damonsson on Slack.

Reply:
We always use only Sylius' copyright information in the project, for example, here is a typical Kernel.php from Symfony 1:1: https://github.com/Sylius/Sylius/blob/2.1/src/Kernel.php

But it could be a great idea to include such information and acknowledgments in the mian README, as it would be more visible to everyone.